### ISSUE TITLE
Report task details in QUERY_ALL_TASKS command

### ISSUE THREAD

**Issue by jphickey** [nasa/cFE#979]
**Is your feature request related to a problem? Please describe.**
The current `CFE_ES_TaskInfo` structure, which is written by the QUERY_ALL_TASKS command, only has some very basic task information - the name/ID, execution count, and what app the task is associated with.

**Describe the solution you'd like**
Suggest to also include:
- stack size
- priority
- entry point? (would depend on #978)

**Additional context**
stack size is an important statistic for determining memory usage
priority is potentially important for real time concerns
Both of these are tracked by OSAL and easily available, just have to get it and put it in the structure.

**Requester Info**
Joseph Hickey, Vantage Systems, Inc.

**Comment by jphickey** [comment]
Question for @acudmore, @jwilmot, @skliper -- this is pretty easy to add if you concur its worthwhile.

**Comment by skliper** [comment]
Good idea but not a priority.
